src/main/java/ezschedule/ui/CalendarBox.java

```diff
@@ -84,24 +84,26 @@ private void setToday(boolean isToday) {
         }
     }
 
+    private void setDate(String date) {
+        calendarDate.setText(date);
+    }
+
     private void setEvents() {
         if (events != null) {
-            int firstEvent = 0;
-            String eventNameOne = getEventName(events.get(firstEvent));
-            Label eventOne = new Label(eventNameOne);
-            setEventLabelStyle(eventOne);
-            calendarEvents.getChildren().add(eventOne);
+            int firstEventIndex = 0;
+            Event firstEvent = events.get(firstEventIndex);
+            String firstEventName = getEventName(firstEvent);
+            setEventLabel(firstEventName);
 
             if (events.size() == 2) {
-                int secondEvent = 0;
-                String eventNameTwo = getEventName(events.get(secondEvent));
-                Label eventTwo = new Label(eventNameTwo);
-                setEventLabelStyle(eventTwo);
-                calendarEvents.getChildren().add(eventTwo);
+                int secondEventIndex = 1;
+                Event secondEvent = events.get(secondEventIndex);
+                String secondEventName = getEventName(secondEvent);
+                setEventLabel(secondEventName);
+
             } else if (events.size() > 2) {
-                Label moreEvents = new Label("...");
-                setEventLabelStyle(moreEvents);
-                calendarEvents.getChildren().add(moreEvents);
+                String moreEvents = "...";
+                setEventLabel(moreEvents);
             }
         }
     }
```

Comment:
Would it be better to use [guard clauses](https://medium.com/@scadge/if-statements-design-guard-clauses-might-be-all-you-need-67219a1a981a) here instead, to reduce the code indentation, making it easier to read?